bw2analyzer/lci.py

```diff
@@ -1,14 +1,60 @@
 import pandas as pd
-from bw2data import get_activity
+from bw2data import Database
 from bw2calc import LCA
+from typing import List, Union
 
 
-def get_labeled_inventory(lca: LCA) -> pd.DataFrame:
+def _get_dependent_database_names(lca: LCA) -> List[str]:
+    databases = []
+    for key in lca.remapping_dicts.keys():
+        databases += [key[0] for key in lca.remapping_dicts[key].values()]
+    return pd.unique(databases)
+
+
+def _load_database_metadata(
+    lca: LCA, cols: Union[None, List[str], str] = None
+) -> pd.DataFrame:
+    # get database names
+    databases = _get_dependent_database_names(lca)
+    # load metadata and store in one dataframe
+    df = pd.DataFrame()
+    for db in databases:
+        df = df.append(pd.DataFrame(Database(db)), ignore_index=True)
+    # drop unwanted columns
+    if cols is None:
+        cols = [
+            "id",
+            "name",
+            "location",
+            "reference product",
+            "type",
+            "unit",
+            "database",
+            "categories",
+        ]
+    elif cols == "all":
+        cols = df.columns.tolist()
+    existing_cols = [c for c in cols if c in df]
+    df = df[existing_cols]
+    # set index
+    df = df.set_index("id")
+    return df
+
+
+def get_labeled_inventory(
+    lca: LCA,
+    wide_format: bool = True,
+    usecols: Union[None, List[str], str] = None,
+) -> Union[pd.DataFrame, pd.Series]:
     """
     Take an LCA's inventory matrix and labels its rows (biosphere) and columns (technosphere) with activity metadata.
 
     Args:
         * *lca* (bw2calc.LCA): LCA object whose life cycle inventory has been calculated previously.
+        * *wide_format* (bool): Whether to return a pd.DataFrame in wide format (biosphere x technosphere)
+            or a pd.Series in long format ((biosphere x technosphere) x 1)
```

Comment:
Long is also called tidy in the R world, at least. Documentation here should be clearer, I can imagine what this means but am not sure.

Reply:
Okay. Do you think it would help to refer to this wikipedia article? https://en.wikipedia.org/wiki/Wide_and_narrow_data

Reply:
I think this is fine for the code, but the docs should have a nice presentation of the dataframe with the indices in pretty html.